app/api/email_utils.py

```diff
@@ -74,3 +74,19 @@ def send_email_verification_message(user_name, email):
     html = render_template('email_confirmation.html', confirm_url=confirm_url, user_name=user_name)
     subject = "Mentorship System - Please confirm your email"
     send_email(email, subject, html)
+
+
+def send_email_about_request(user_name, email):
+    """Sends an html email message to the specified user.
+
+        Args:
+            user_name: User's name.
+            email: User email address.
+        """
+    confirmation_token = generate_confirmation_token(email)
+    from app.api.resources.user import UserEmailConfirmation  # import here to avoid circular imports
+    from app.api.api_extension import api
+    confirm_url = api.url_for(UserEmailConfirmation, token=confirmation_token, _external=True)
+    html = render_template('email_new_request_is_received.html', confirm_url=confirm_url, user_name=user_name)
```

Comment:
If this email template does use external parameters inside then you don't need to pass the confirmation token :)
Although the user name can stay, since its being used inside the template.

So here you can delete line 86, 87, 88, 89 since they are not needed for this new function.